components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -558,6 +850,96 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
                 />
               </div>
             </div>
+            <h1 className="text-md font-semibold text-slate-900 my-2">Highlight suggestions</h1>
+            <Swiper
+              spaceBetween={8}
```

Comment:
I get the same Netlify error on running `npm run prebuild && npm run build`:

```
info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
- info Linting and checking validity of types ...Failed to compile.

./components/molecules/HighlightInput/highlight-input-form.tsx:627:17
Type error: Type '{ children: Element[]; spaceBetween: number; slidesPerView: number; className: string; modules: any[]; pagination: { clickable: boolean; }; a11y: { enabled: boolean; }; }' is not assignable to type 'IntrinsicAttributes & RefAttributes<SwiperRef> & SwiperProps'.
  Property 'spaceBetween' does not exist on type 'IntrinsicAttributes & RefAttributes<SwiperRef> & SwiperProps'.

  625 |             ) : (
  626 |               <Swiper
> 627 |                 spaceBetween={8}
      |                 ^
  628 |                 slidesPerView={1}
  629 |                 className="lg:w-[33vw] md:w-[50vw]"
  630 |                 modules={[Pagination, A11y]}
```

My node environment:
```
❯ npm --version
npm info using npm@9.6.7
npm info using node@v18.17.1
9.6.7
npm info ok
```

I do notice after running those build steps I get the following diff in the shrinkwrap which might be irrelivant:
```diff
diff --git a/npm-shrinkwrap.json b/npm-shrinkwrap.json
index 1bcc45a..d155c00 100644
--- a/npm-shrinkwrap.json
+++ b/npm-shrinkwrap.json
@@ -118,8 +118,8 @@
         "typesync": "^0.10.0"
       },
       "engines": {
-        "node": ">= 16.7.0",
-        "npm": ">= 8.0.0"
+        "node": "^18.17.0 || >=20.5.0",
+        "npm": ">= 9.6.0"
       }
     },
     "node_modules/@adobe/css-tools": {
```

Looks like there are similar errors people report with using an older version of TypeScript:
https://github.com/nolimits4web/swiper/issues/6843#issuecomment-1634818157

I'm guessing it would be a pretty huge lift to upgrade our typescript dependency:
https://github.com/open-sauced/app/blob/b1782cb094ff687296324f3354704c6681520d44/package.json#L152

Reply:
We should be able to upgrade the TypeScript dependency without much of a lift in separate PR.

Reply:
Looks like the last deploy still fails with this warning:
https://app.netlify.com/sites/oss-insights/deploys/64fb63ae7fde590008a744ae

Should we try upgrading TS as part of this PR if we need the swiper component?